addons/web/views/report_templates.xml

```diff
@@ -189,15 +189,15 @@
                        </table>
                        <div class="clearfix">
                            <div class="row">
-                               <div class="col-md-6">
+                               <div class="col-6 position-absolute">
                                    <p name="payment_term">
                                        <span>Payment terms: 30 Days</span>
                                    </p>
                                    <p>
                                        Payment Communication: <b><span>INV/2023/00003</span></b>
                                    </p>
                                </div>
-                               <div id="total" class="col-md-6">
+                               <div id="total" class="col-6 ms-auto">
```

Comment:
isn't it enough to set `col-6` classname on both divs?

Reply:
@aab-odoo 
You're right it is, I updated the PR